### PR TITLE
Practice Review

### PR DESCRIPTION
**Pull request by Rwsmith8692**

### CODE REVIEW

File: Practice Review/Practice Review/Practice Review/Program.cs [hunk 1]
```diff
@@ -0,0 +1,108 @@
+﻿using System;
+
+namespace Practice_Review
+{
+    class Program
+    {
+        private const string V = "35";
```
Comment (imazizahurt) [inline Practice Review/Practice Review/Practice Review/Program.cs:7]: the private const string is only needed if you plan to use the value.

File: Practice Review/Practice Review/Practice Review/Program.cs [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+            }
+            else if (myValue < myDouble)
+            {
+                Console.WriteLine("Less than.");
+
+            }
+            else
+            {
+                Console.WriteLine("Equal to.");
+            }
```
Comment (imazizahurt) [inline Practice Review/Practice Review/Practice Review/Program.cs:65]: Good job re-using the variable values for the if, else if and else statements

**Comment by Rwsmith8692** [comment]
Thank you. My girlfriend helped me with the review. She gave me some
pointers and how to do things while I typed everything.

**Review by imazizahurt — APPROVED** [review]
Overall good work! Make sure that you fully understand the functionality of what you're using in order to optimize performance.

**Comment by Rwsmith8692** [comment]
I will. I feel like I'm slowly getting there. Just need to practice more.

**Pull request merged**